ui/core.js

```diff
@@ -13,6 +13,8 @@
 //>>description: The core of jQuery UI, required for all interactions and widgets.
 //>>docs: http://api.jqueryui.com/category/ui-core/
 //>>demos: http://jqueryui.com/
+//>>css.structure: ../themes/base/core.css
```

Comment:
+1 for option 1

I don't like option 2 becuase its too much "magic" to figure out how the file is actually being included.

1 is the simplest, easiest to understand, and easiest to implement. Seems like a clear winner to me for now.